lib/bolt/transport/ssh.rb

```diff
@@ -102,6 +102,11 @@ def initialize
 
         require 'net/ssh'
         require 'net/scp'
+        begin
+          require 'net/ssh/krb'
```

Comment:
We just need the `require 'net/ssh/krb'. Before, when we were not shipping the kerberos gem we would *only* require that library when a user manually installed the gem on their system. Now we have added the kerberos gem to the gemspec and started shipping it in the system packages we build and therefore it will always be available. So we do not need to rescue the `LoadError` anymore. That is what https://github.com/puppetlabs/bolt/pull/1586#pullrequestreview-354130026 was meant to address.

Reply:
ack